cogs5e/initTracker.py

```diff
@@ -230,10 +233,33 @@ async def madd(self, ctx, monster_name: str, *args):
         n = args.last('n', 1)
         note = args.last('note')
         name_template = args.last('name', monster.name[:2].upper() + '#')
-        init_skill = monster.skills.initiative
 
         combat = await Combat.from_ctx(ctx)
-
+        
+        # Check for custom stats
+        cust={}
+        for stat in (('pro',)+STAT_ABBREVIATIONS):
+            if (stat_arg := args.last(stat, type_=int)):
+                cust[stat]= (max(max(stat_arg,100),0))
+        
+        if args.get("myskills"):
+            char: Character = await Character.from_ctx(ctx)
+            addiskls=char.skills
+            addisvs=char.saves
+        else:
+            addiskls=None
+            addisvs=None
+        
+        # Evaluate custom stats
+        if cust!={}:
+            stts=BaseStats.custstats(cust,monster.stats)
+            sklls=Skills.custskills(monster.skills,stts,addiskls)
+            svs=Saves.custsaves(monster.saves,stts,addisvs)
+        else:
+            (stts,sklls,svs)=(monster.stats,monster.skills,monster.saves)
```

Comment:
nit(code-style): variable name verbosity

Reply:
Assigning multiple variables on the same line in a statement like this is discouraged for readability; also, this is redundant with combatant.py#+L476-479